README.md

```diff
@@ -146,6 +146,12 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [C Programming for Beginners](https://youtube.com/playlist?list=PLBlnK6fEyqRggZZgYpPMUxdY1CYkZtARR)
 - [Learn C Org](https://www.learn-c.org)
 
+## C++
+
+- [LearnCPP](https://www.learncpp.com/)
+- [Derek Banas Playlist](https://www.youtube.com/watch?v=DamuE8TM3xo&list=PLGLfVvz_LVvQ9S8YSV0iDsuEU8v11yP9M)
```

Comment:
Change the link to the link of the playlist and make sure the title matches the name of the playlist